src/org/scratchOrgSettingsGenerator.ts

```diff
@@ -82,13 +86,19 @@ export const createObjectFileContent = ({
   return { ...output, ...{ version: apiVersion } };
 };
 
-const calculateBusinessProcess = (objectName: string, defaultRecordType: string): Array<string | null> => {
+const calculateBusinessProcess = (
+  objectName: string,
+  defaultRecordType: string,
+  capitalizeBusinessProcess: boolean
+): Array<string | null> => {
   let businessProcessName = null;
   let businessProcessPicklistVal = null;
   // These four objects require any record type to specify a "business process"--
   // a restricted set of items from a standard picklist on the object.
   if (['Case', 'Lead', 'Opportunity', 'Solution'].includes(objectName)) {
-    businessProcessName = upperFirst(defaultRecordType) + 'Process';
+    businessProcessName = capitalizeBusinessProcess
+      ? `${upperFirst(defaultRecordType)}Process`
+      : `${defaultRecordType}Process`;
```

Comment:
this wasn't in the scope of the inv. we got but it's the same behavior (for the listed 4 objects above) so I made this follow the config var too.